src/styles/blocks/fonts.css

```diff
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Book.ttf);
+}
```

Comment:
Using the same `font-family` name 'Avenir' for different font files can lead to conflicts. Consider specifying different `font-weight` or `font-style` properties for each `@font-face` rule to differentiate between the font files.